themes/marvelous-snuffle/static/css/style.css

```diff
@@ -1,39 +1,51 @@
 /* ============================================================= COLORS ** ============================================================= */
+:root{
+    --global-font-color:  #e0e0e0;
+    --global-button-background-color: #283747;
+    --background-color-2: #212121;
+    --global-border-color: var(--global-font-color);
+    --global-border: 1px solid var(--global-border-color);
+    --global-font-size: 13px;
+    --global-padding: 15px;
+    --fill-color: var(--global-button-background-color);
+    --active-color: #0c99
```

Comment:
this should be `primary` color, we can also add `secondary` and other colors this way.
please look at https://github.com/twbs/bootstrap/blob/main/scss/_variables.scss#L69

Reply:
ok, thx